### PR TITLE
event rounded corners

### PR DESCRIPTION
**Pull request by rjain90**
Fix for #843

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -4,9 +4,9 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'de.timfreiheit.resourceplaceholders'
 
-def keystorePropertiesFile = rootProject.file("keystore.properties")
```
Comment (tibbi) [inline app/build.gradle]: even if you did these changes locally, you definitely shouldnt have commited them. Revert it please.

File: app/src/main/kotlin/com/simplemobiletools/calendar/pro/activities/SimpleActivity.kt [hunk 2]
```diff
@@ -68,4 +68,6 @@ open class SimpleActivity : BaseSimpleActivity() {
     private fun unregisterObserver() {
         contentResolver.unregisterContentObserver(calDAVSyncObserver)
     }
+
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/calendar/pro/activities/SimpleActivity.kt]: these extra lines also arent needed

**Comment by tibbi** [comment]
creating a new pull request might be the easiest way of getting rid of those unwanted changes. Thanks

File: app/build.gradle (hunk 1 above)
Comment (rjain90) [reply app/build.gradle]: This file is not in the master branch. In order to build the code, I'm assuming everyone has to comment these lines.

Comment (tibbi) [reply app/build.gradle]: you shouldnt care about everyone else, only about the issue you are fixing right at the moment

Comment (rjain90) [reply app/build.gradle]: - Everyone includes me. How would I build the code successfully without the "keystore.properties" file?

Comment (tibbi) [reply app/build.gradle]: you can change any code locally, but you shouldnt push it to the master branch

**Pull request closed without merging**